docs/docs/dev_docs/getting_started/cli.md

```diff
@@ -4,11 +4,13 @@ title: Aztec CLI
 
 ## Introduction
 
-The Aztec CLI is a tool designed to enable a user to interact with the Aztec Network. Here we will provide a walk-through demonstrating how to use the CLI to deploy and test contracts on the [Aztec Sandbox](../sandbox/main.md).
```

Comment:
> Here we will provide a walk-through demonstrating how to use the CLI to deploy and test contracts on the Aztec Sandbox

I think it's still beneficial to explain the goal of this page at the beginning.

Reply:
agree. I will move it back up